modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/js/AccessibilityMenu.tsx

```diff
@@ -97,7 +122,7 @@ const AccessibilityMenu = (props) => {
 				})
 			);
 		},
-		[setSettings]
+		[settings, setSettings]
```

Comment:
What's the reason for adding this to the dependencies of this hook, instead of passing a function to `setSettings` that can access the current value of settings directly? Wouldn't this result in infinite re-renders since `settings` would change every time we call `setSettings`?

Reply:
@ethib137 I thought this was working but the set state of `useLiferayState` doesn't support function so it wasn't working before. But in this case it won't cause an infinite loop because this method is not used as a dependency on `useEffect` for example.